.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
           - --skip="./.*,*.csv,*.json"
           - --quiet-level=2
         exclude_types: [csv, json]
-        exclude: ^tests/fixtures/|homeassistant/generated/
+        exclude: ^tests/fixtures/|homeassistant/generated/|sentences/
```

Comment:
We should package them and not copy them into core at all.

Reply:
This will be next :+1:

Reply:
https://pypi.org/project/home-assistant-intents/